### PR TITLE
Kevm cleanup

### PR DESCRIPTION
**Pull request by ehildenb**
Various fixes to KEVM script, including fixes which allow running `kevm` script from a different directory.

### CODE REVIEW

**Comment by ehildenb** [comment]
Jenkins: test this please

**Comment by ehildenb** [comment]
Jenkins: test this please

File: Makefile [hunk 1]
```diff
@@ -172,36 +172,39 @@ slow_bchain_tests=$(wildcard tests/ethereum-tests/BlockchainTests/GeneralStateTe
...
 proof_tests=$(wildcard $(proof_dir)/*/*-spec.k)
 
-proof-test-all: proof-test
-proof-test: $(proof_tests:=.test)
+test-all-proof: test-proof
+test-proof: $(proof_tests:=.test)
 
 $(proof_dir)/%.test: $(proof_dir)/% build-java
-	$(TEST) $<
+	MODE=NORMAL SCHEDULE=DEFAULT $(TEST) $< tests/templates/proof-success.txt
```
Comment (daejunpark) [inline Makefile]: not sure if `SCHEDULE=DEFAULT` matters, but the proof spec admits the BYZANTIUM schedule.

**Comment by daejunpark** [comment]
Jenkins: test this please

File: Makefile (hunk 1 above)
Comment (daejunpark) [reply Makefile]: I think it is better to drop this, since kevm has the default value.

File: kevm [hunk 2]
```diff
@@ -3,6 +3,19 @@
 set -e      # Exit immediately if any command fails
 set -u      # Using undefined variables is an error. Exit immediately
 
+kevm_script="$0"
+while [[ -h "$kevm_script" ]]; do
+    kevm_dir="$(cd -P "$(dirname "$kevm_script")" && pwd)"
+    kevm_script="$(readlink "$kevm_script")"
+    [[ "$kevm_script" != /* ]] && kevm_script="$kevm_dir/$kevm_script"
+done
```
Comment (daejunpark) [inline kevm]: impressive :)

**Comment by daejunpark** [comment]
@ehildenb looks good!

**Pull request merged**